vm/src/stdlib/io.rs

```diff
@@ -178,20 +178,20 @@ impl BufferedIO {
 }
 
 #[derive(Debug)]
-struct PyStringIO {
```

Comment:
I'm wondering whether we should keep this as `PyStringIO` as it's called `StringIO` in Python.